tensorflow/compiler/xla/service/cpu/runtime_matmul_mkl.h

```diff
@@ -19,24 +19,16 @@ limitations under the License.
 #include <iostream>
 
 #ifdef ENABLE_MKL
-#include "third_party/intel_mkl_ml/include/mkl_cblas.h"
+#include "dnnl.hpp"
```

Comment:
Could you please use the full path to the header file from now on? Thank you!

Reply:
Sure, I will make the change. Thanks